tests/asrc_test/dut/with_xscope_fileio/src/file_utils/fileio.c

```diff
@@ -0,0 +1,92 @@
+// Copyright 2024 XMOS LIMITED.
```

Comment:
Should this file be included? It is part of xscope_fileio no?

Reply:
No, this is a wrapper on top of xscope_fileio API. If compiled without TEST_WAV_XSCOPE, it calls Posix fileio functions instead of xscope_fileio. The idea is to have the app run for x86 as well (not tested!). We used it in fwk_voice test_wav type tests and I reused it here.

Reply:
Ah, fair play. I it looked quite familiar but I didn't look in detail. If useful, perhaps an item for xscope_fileio at some point.